prody/utilities/catchall.py

```diff
@@ -63,6 +63,8 @@ def calcModeClusters(similarities, n_clusters=0, linkage='all', method='tsne', c
         
 
     # Check inputs to make sure are of valid types/values
+    from prody.utilities import isListLike
```

Comment:
This should be added to line 6 not here. You don't want to import `prody.utilities` inside `prody.utilities.catchall` too. `.misctools` means `misctools` from the same module (`prody.utilities.misctools`).

Reply:
You can see #1085 where I've done that

Reply:
Ok this is good now